lib/config/rules/stylistic-issues.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
   'brace-style': ['warn', '1tbs', {allowSingleLine: true}],
   // Require camel case names
   'camelcase': ['warn', {properties: 'always'}],
+  // Enforce or disallow capitalization of the first letter of a comment
+  'capitalized-comments': 'off',
```

Comment:
Initially, I had this enabled but then turned it off after thinking the same thing.